src/main/java/seedu/realodex/model/person/Income.java

```diff
@@ -12,6 +12,7 @@ public class Income {
     /** Message for constraints on income. */
     public static final String MESSAGE_CONSTRAINTS = "Income should be an integer and should be at least 0";
     public static final String VALIDATION_REGEX = "^[0-9]+$";
+    public static final String DEFAULT_INCOME = "1";
```

Comment:
```suggestion
    public static final String DEFAULT_INCOME = "0";
```
Set default to 0 might make more sense